src/main/webapp/app/shared/components/clone-repo-button/clone-repo-button.component.html

```diff
@@ -26,13 +26,19 @@
                     <a (click)="setUseSSH(true)" class="dropdown-item {{ useSsh ? 'active' : '' }}" id="useSSHButton">SSH</a>
                 </div>
             </div>
+            <!-- ngStyle: sshEnabled -> Remove round edges and edge on the left side || useSsh -> Remove round edges and edge on the right side -->
             <pre
                 style="flex: 1"
-                [ngStyle]="sshEnabled ? { borderTopLeftRadius: 0, borderBottomLeftRadius: 0, borderLeft: 'none' } : {}"
+                [ngClass]="sshEnabled && !useSsh ? 'remove-line-left-right' : sshEnabled && useSsh ? 'remove-line-left' : 'remove-line-right'"
```

Comment:
You can replace `remove-line-left-right` with `remove-line-left remove-line-right`. And parentheses for nested ternary operators make it more readable :)